src/Switch-Arduino_MCP4728/main.py

```diff
@@ -168,17 +206,20 @@ def unconfigure(self):
         else:
             self.set_voltage(self.pin, 0)
 
-    def apply(self):
+    def apply(self) -> None:
+        """Set the voltages to the pins."""
         if self.multi_pins:
             # Receive values as list, split by ":" and convert to float
             split_values = self.value.split(":")
+            # Replace , with . to enable float conversion
+            split_values = split_values.replace(",", ".")
```

Comment:
it must be added above the split. Otherwise it must be done elementwise.

Although I did it myself during my quick fix, could you implement it in a way that self.value is not overwritten. We had once a case where this lead to trouble, e.g. using

sweepvalue = self.value.replace(",", ".")
split_values = sweepvalue.split(":")